nuxt.config.ts

```diff
@@ -29,11 +28,22 @@ export default defineNuxtConfig({
       apiUrl: process.env.NUXT_API_URL,
     },
   },
+  gtm: {
+    id: process.env.NUXT_APP_GTM_ID || '',
+    enabled: process.env.NUXT_APP_GTM_ID === 'true' ? true : false, // Enable GTM tracking
```

Comment:
@igor-kamil could not come up with anything better

Reply:
```suggestion
    enabled: process.env.NUXT_APP_GTM_ENABLED === 'true' ? true : false, // Enable GTM tracking
```
would be better 😂 

however `enabled` doesn't seems to really work anyway. I've found this [workaround](https://github.com/zadigetvoltaire/nuxt-gtm/issues/26#issuecomment-2190851752)